developer-extension/src/panel/components/json.tsx

```diff
@@ -185,7 +263,7 @@ function JsonValue({ descriptor }: { descriptor: JsonValueDescriptor }) {
               parentType: 'array',
               parentValue: descriptor.value as unknown[],
               value: child,
-              path: descriptor.path,
+              path: descriptor.path ? `${descriptor.path}.${i}` : String(i),
```

Comment:
Issue: The omission of the index was intentional. It follows the events platform query, so you can copy a path and paste it in the Datadog ui.